examples/Bremsstrahlung/cmakeFlags

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env bash
+#
+# Copyright 2013-2017 Heiko Burau, Axel Huebl, Rene Widera
+#
+# This file is part of PIConGPU.
+#
+# PIConGPU is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# PIConGPU is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with PIConGPU.
+# If not, see <http://www.gnu.org/licenses/>.
+#
+
+#
+# generic compile options
+#
+
+################################################################################
+# add presets here
+#   - default: index 0
+#   - start with zero index
+#   - increase by 1, no gaps
+
+flags[0]="-DCUDA_ARCH=20 -DPARAM_OVERWRITES:LIST=-DENABLE_BREMSSTRAHLUNG=1;"
```

Comment:
please call such test-case controls `PARAM_`, so namely: `PARAM_BREMSSTRAHLUNG=1`.
last argument without `;`.

in your case, bremsstrahlung shall be on by default, so just use *no defines at all* in this example and have it always on :)